backend/src/data/models/user.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  ModelDefined,
+  DataTypes,
+  Optional
+} from "sequelize";
+
+import { sequelize } from '../db/connection';
+import { User } from '../../../../shared/src/interfaces/user.interface';
+
+interface UserCreationAttributes extends Optional<User, "id"> {}
+
+const UserModel: ModelDefined<
```

Comment:
See the https://github.com/BinaryStudioAcademy/bsa-2021-healthcare/pull/2 PR and do model like there (with all my remarks)